### PR TITLE
Issue996 hydronictest 

### PR DESCRIPTION
**Pull request by JavierArroyoBastida**

### CODE REVIEW

**Comment by JavierArroyoBastida** [comment]
@Mathadon please do not merge yet as there are still some pending issues that need to be updated. I'll let you know once ready.

**Comment by JavierArroyoBastida** [comment]
@Mathadon could you please review?

File: IDEAS/Examples/IBPSA/SingleZoneResidentialHydronic.mo [hunk 1]
```diff
@@ -1,12 +1,17 @@
 within IDEAS.Examples.IBPSA;
 model SingleZoneResidentialHydronic
   "Single zone residential hydronic example model"
+  import IBPSA;
```
Comment (Mathadon) [inline IDEAS/Examples/IBPSA/SingleZoneResidentialHydronic.mo:4]: We should not need this import as we should only use IDEAS models.

Comment (JavierArroyoBastida) [reply IDEAS/Examples/IBPSA/SingleZoneResidentialHydronic.mo]: Ups... https://github.com/open-ideas/IDEAS/pull/1143/commits/8c7e1b16ccbc9b4e5389dfce926faa016619bb54

**Pull request merged**